packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -651,7 +652,7 @@ export class GarbageCollector implements IGarbageCollector {
      * We current write the entire GC state in a single blob. This can be modified later to write multiple
      * blobs. All the blob keys should start with `gcBlobPrefix`.
      */
-    public summarize(): ISummaryTreeWithStats | undefined {
+    public summarize(telemetryContext?: ITelemetryContext): ISummaryTreeWithStats | undefined {
         if (!this.shouldRunGC || this.previousGCDataFromLastRun === undefined) {
             return;
         }
```

Comment:
It would be great to capture GC stats in telemetryContext below and get rid of gcBlobNodeCount, gcTotalBlobsSize (see IGeneratedSummaryStats). I.e. use more generic mechanism you are adding and move logic to the right layer (I'd think that's the better place to collect that data).
This work, however, can be done later and pushed to GC feature crew (FC). Maybe worth opening a task and assign to Tyler.